extensions/pkg/webhook/webhook.go

```diff
@@ -59,24 +64,34 @@ type Args struct {
 	Provider   string
 	Name       string
 	Path       string
+	Target     string
 	Predicates []predicate.Predicate
 	Validators map[Validator][]Type
 	Mutators   map[Mutator][]Type
 }
 
 // New creates a new Webhook with the given args.
 func New(mgr manager.Manager, args Args) (*Webhook, error) {
-	logger := log.Log.WithName(args.Name).WithValues("provider", args.Provider)
+	var (
+		objTypes   []Type
+		actionType string
 
-	// Create handler
-	builder := NewBuilder(mgr, logger)
+		logger  = log.Log.WithName(args.Name).WithValues("provider", args.Provider)
+		builder = NewBuilder(mgr, logger)
+	)
 
 	for val, objs := range args.Validators {
 		builder.WithValidator(val, objs...)
+		objTypes = append(objTypes, objs...)
+		if actionType != ActionMutating {
+			actionType = ActionValidating
+		}
```

Comment:
What is the meaning of this if statement? Isn't its condition always true that `actionType != ActionMutating` because at this point `actionType` is an unassigned local variable with value ""